include/aspect/simulator_access.h

```diff
@@ -781,14 +781,32 @@ namespace aspect
        * constraints for the time step we are currently solving.
        */
       const ConstraintMatrix &
-      get_current_constraints() const;
+      get_current_constraints () const;
 
       /**
        * Return whether or not the current object has been initialized by providing it with
        * a pointer to a Simulator class object.
        */
       bool simulator_is_initialized () const;
 
+      /**
+       * Return whether the Simulator object has been completely initialized
+       * and has started to run its time stepping loop. Obviously, for this
+       * function to work, the simulator_is_initialized() function needs
+       * to return @p true.
+       *
+       * This function is useful to determine in a plugin whether some
+       * of the information one can query about the Simulator is reliable
+       * or has already been set. For example, while the Simulator is being
+       * set up, plugins may already have access to it via the current
+       * SimulatorAccess object, but data such as the current time, the
+       * time step number, etc, may all still be in a state that is not
+       * reliable since it may not have been initialized at that time.
+       * Only when this function returns @p true is all of the information
+       * returned by the SimulatorAccess object reliable and correct.
```

Comment:
you never mention that this will be false during mesh refinement later in the simulation, can you add a sentence?